packages/snage/package.json

```diff
@@ -54,6 +53,7 @@
     "npm-run-all": "^4.1.5",
     "portable-fetch": "^3.0.0",
     "prettier": "^1.18.2",
+    "snage-ui": "*",
```

Comment:
Wie funktioniert das überhaupt? `snage-ui` ist kein npm-Module, richtig? yarn/lerna checkt automatisch, dass das in einem parallelen Package liegt und nimmt immer direkt die Sourcen? Oder könnte yarn auch was gecacht aus `node_modules` ziehen?

Reply:
genau, lerna/yarn linkt dan korrekt